docs/guidelines.md

```diff
@@ -7,15 +7,13 @@ more about the [goal of exercism here](https://github.com/exercism/docs/blob/mas
 
 ## Definitions
 
-In the following paragraphs, keywords such as **MUST**, **SHOULD**, **COULD**
+In the following paragraphs, keywords such as **MUST**, **SHOULD**, **MAY**
```

Comment:
I like using `MAY`! The three values now a really distinct from each other (instead of only differing by one or two characters).

Reply:
And #58 was totally right! I got COULD from MoSCow, **MAY** is the official one!